build/Makefile

```diff
@@ -317,7 +317,7 @@ install: FEATURE_GATES ?= $(ALPHA_FEATURE_GATES)
 install: HELM_ARGS ?=
 install: $(ensure-build-image) install-custom-pull-secret
 	$(DOCKER_RUN) \
-		helm upgrade --install --wait --namespace=agones-system \
+		helm upgrade --install --atomic --namespace=agones-system \
```

Comment:
The helm docs answered my question: `The --wait flag will be set automatically if --atomic is used`